webdriver/tests/perform_actions/wheel.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from webdriver.error import NoSuchWindowException
+
+from tests.perform_actions.support.refine import filter_dict, get_events
+from tests.support.asserts import assert_move_to_coordinates
+
+
+def test_null_response_value(session, wheel_chain):
+    value = wheel_chain.scroll(0, 0, 0, 10).perform()
+    assert value is None
+
+
+def test_no_browsing_context(session, closed_window, wheel_chain):
+    with pytest.raises(NoSuchWindowException):
+        wheel_chain.scroll(0, 0, 0, 10).perform()
+
+
+def test_wheel_scroll(session, test_actions_page, wheel_chain):
```

Comment:
Would be good to also have a test for scrolling something other than the main document e.g. an iframe or something with overflow: scroll.